packages/contracts/contracts/L1/AssetManager.sol

```diff
@@ -77,15 +72,9 @@ contract AssetManager is ISemver, IERC721Receiver, IAssetManager {
     uint256 public immutable UNDELEGATION_PERIOD;
 
     /**
-     * @notice The numerator of the slashing rate.
+     * @notice The amount to slash.
      */
-    uint128 public immutable SLASHING_RATE;
-
-    /**
-     * @notice Minimum amount to slash. It should be equal or less than
-     *         ValidatorManager.MIN_ACTIVATE_AMOUNT.
-     */
-    uint128 public immutable MIN_SLASHING_AMOUNT;
+    uint128 public immutable SLASHING_AMOUNT;
```

Comment:
Why don't we name this, `BOND_AMOUNT`?

Reply:
I also it would be better for code readability